src/components/KTableView/KTableView.vue

```diff
@@ -963,11 +812,11 @@ const startResize = (evt: MouseEvent, colKey: string) => {
 }
 
 const showPagination = computed((): boolean => {
-  if (props.hidePagination) {
+  if (props.hidePagination || props.nested) {
```

Comment:
There appear to be changes unrelated to KTableData in this PR as well? So now we'll need to test _all_ of the added functionality?

Reply:
Wdym? Only functionality added to KTableView in this PR is `hidePaginationWhenOptional` prop, which adds minor change and has been tested. All other changes are either refactor related to props handling (`withDefaults`) or minor fixes found during KTableData adoption (such as this line).

Reply:
> minor fixes found during KTableData adoption

This explains what I was asking 👍🏼